a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -167,7 +167,7 @@ <h3>Terminology</h3>
 <dt><dfn>electronic braille</dfn></dt>
```

Comment:
If we don't use a term in a document, we can't define it in the terminology. If you want to compare the terms in the definition, that's fine, but you need to remove the one that isn't being used in the documents. That would be electronic braille now, from what I can tell.